msg/battery_status.msg

```diff
@@ -10,8 +10,12 @@ float32 scale		# Power scaling factor, >= 1, or -1 if unknown
 float32 temperature         # temperature of the battery. NaN if unknown
 int32 cell_count		# Number of cells
 bool connected			# Whether or not a battery is connected, based on a voltage threshold
-bool system_source		# Whether or not a this battery is the active power source for VDD_5V_IN
-uint8 priority        	# Zerro based priority is the connection on the Power Controller V1..Vn AKA BrickN-1
+
+uint8 BATTERY_SOURCE_POWER_MODULE = 0
+uint8 BATTERY_SOURCE_EXTERNAL = 0
+uint8 BATTERY_SOURCE_ESCS = 0
```

Comment:
0, 1, 2 ?

Reply:
I don't see where the BAT%d_SOURCE parameter definitions are located? Could you point me to them and make sure the documentation is updated?

Reply:
Oops!

Reply:
Fixed in a0c82b1.